File: samples-vb/getting-started/BinaryClassification_SentimentAnalysis/TestSentimentData.vb
Friend Class TestSentimentData
    Friend Shared ReadOnly Sentiments As IEnumerable(Of SentimentData) = {
        New SentimentData With {
            .SentimentText = "Contoso's 11 is a wonderful experience",
            .Sentiment = 0
        }, New SentimentData With {
            .SentimentText = "The acting in this movie is very bad",
            .Sentiment = 0
        }, New SentimentData With {
            .SentimentText = "Joe versus the Volcano Coffee Company is a great film.",
            .Sentiment = 0
        }
    }
End Class

File: samples/csharp/getting-started/BinaryClassification_SentimentAnalysis/TestSentimentData.cs
using System.Collections.Generic;

namespace BinaryClassification_SentimentAnalysis
{
    internal class TestSentimentData
    {
        internal static readonly IEnumerable<SentimentIssue> Sentiments = new[]
        {
            new SentimentIssue
            {
                text = "Contoso's 11 is a wonderful experience",
                label = 0
            },
            new SentimentIssue
            {
                text = "The acting in this movie is very bad",
                label = 0
            },
            new SentimentIssue
            {
                text = "Joe versus the Volcano Coffee Company is a great film.",
                label = 0
            }
        };
    }
}